README.rst

```diff
@@ -70,11 +70,26 @@ guide_. If you are a new developer who would like to start contributing to
 MDAnalysis as a start you can increase our code coverage, the guides explain how
 to find uncovered code.
 
+
+
+The MDanalysis project uses an `open governance model` and is fiscally sponsored by `NumFOCUS`_. Consider making 
+a `tax-deductible donation`_ to help the project pay for developer time, professional services, travel, workshops, and a variety of other needs.
+
+.. image:: https://raw.githubusercontent.com/numfocus/templates/master/images/numfocus-logo.png
+  :scale: 1 %
+  :target: <PROJECT_NUMFOCUS_SPONSORSHIP_LINK>
```

Comment:
Do we want to wait with merging until we have the link?

Reply:
probably yep

Reply:
added links etc